docs/trustedsetup.md

```diff
@@ -10,28 +10,25 @@ to manage a large amount of value, and if their users accept that the risk of
 coordinator misbehaviour is insufficient to justify doing the work nof a
 trusted setup. After all, MACI's security model presumes a trusted coordinator.
 
+The PSE team is making available a trusted set of zKeys for MACI's circuits,
+which will be available as soon as the first ceremony is completed and
+will be visible on
+[p0tion's website](https://ceremony.pse.dev/projects/Maci%20v1%20Trusted%20Setup%20Ceremony).
+
 In any case, MACI can be relatively easily modified to support PLONK, which
 does not require a circuit-specific trusted setup. Its circuits, written in
 [`circom`](https://github.com/iden3/circom), are compatible with Fluidex's
 [`plonkit`](https://github.com/Fluidex/plonkit) tool. The downside to using
 PLONK is that proof generation is not as optimised as it is for Groth16.
 
-## How to run the trusted setup
-
-First, follow the instructions in the [Installation](./installation.html)
-section to install dependencies for MACI.
-
-Next, configure and compile circuits following instructions in
-[Circuits](./circuits.html).
-
-Finally, use the [`multisetups`](https://github.com/privacy-scaling-explorations/multisetups)
-tool to do this.
-
-You should perform at least one contribution to each circuit, even if you
-choose not to perform a multi-party trusted setup.
-
-We don't recommend a browser-based trusted setup (which [Tornado
-Cash](https://ceremony.tornado.cash/) and [Zkopru](https://mpc.zkopru.network/)
-used) for the MACI circuits as they are too large to be feasibly processed in
-the browser.
-
+## How to run your own trusted setup
```

Comment:
Why are we quoting this one? Isn't a cross-reference in the paragraph above sufficient?

It seems to me "too trusted setup tooling centric" and that it has little to do with MACI tbh.

Reply:
Ah, in that case even less imho. They could use another tool and there's no point in us telling them how they should launch with p0tion here if we already link above docs. It seems like repetition to me and I would keep the docs concise.